renderer/src/shared/Hint/types.ts

```diff
@@ -0,0 +1,32 @@
+import { ReactNode, MutableRefObject } from "react";
+
+export type Position =
+  | {
+      basePosition: "bottom";
```

Comment:
you didn't change them here (see Oleg's comment)

Reply:
Yep, but types can take only types. It is impossible to pass a constant to a type. We have already discussed this with Oleg

Reply:
agree